packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -129,9 +129,17 @@ class EventDetails extends CoreElement {
   }
 
   Future<void> update(FrameFlameChartItem item) async {
-    ga.select(ga.timeline, ga.timelineFlame);
     _event = item.event;
 
+    String gaTimeLineFlame;
+    if (_event.isGpuEvent) {
+      gaTimeLineFlame = ga.timelineFlameGpu;
+    } else {
+      gaTimeLineFlame = ga.timelineFlameUi;
+    }
+    ga.select(
```

Comment:
I'm still seeing this code in the latest refactor. I expect this line is called 100s of time rendering a frame unless I'm missing something.

Reply:
I'm not seeing that only on user click.

Reply:
This looks right. `update` is only called from `onSelectedFrameFlameChartItem.listen` in timeline.dart:183, but I still think this ga code should live there. I think it makes more sense to call ga.select from the `onSelectedFrameFlameChartItem` listener rather than this side effect method (event_details.update()).

Reply:
Added ga.select call to onSelectedFrameFlameChartItem in timeline.dart